llvm/docs/CommandGuide/llvm-debuginfo-analyzer.rst

```diff
@@ -855,6 +861,73 @@ layout and given the number of matches.
   -----------------------------
   Total           26          8
 
+DEBUGGER VIEW
+"""""""""""""
+In debugger view, :program:`llvm-debuginfo-analyzer` prints out
+debug-info in a manner that emulates a debugger. For each function, each
+statement line is printed out in order, complete with the inlined
+callstack. This is useful to verify the specific orders of lines, as
+well as verifying inline callstacks.
+
+.. code-block:: none
+
+  llvm-debuginfo-analyzer --report=debugger
+                          test-dwarf-clang.o test-dwarf-gcc.o
```

Comment:
For consistency with the another general printing layouts, include:
`Logical View` title follow by `{File` and `CompileUnit` entries

 ```
 Logical View:
  [000]           {File} 'test-dwarf-clang.o'
  [001]           {CompileUnit} 'test.cpp'
```